soc/renesas/ra/ra2l1/ofs_mem.c

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
I don't want to have multiple implementations, can you just do it the way RA4M1 does?
Or, if your implementation is better, can you update RA4M1?

https://github.com/zephyrproject-rtos/zephyr/blob/14b3293d94a1997d0d95dc5b03317c13b3854881/soc/renesas/ra/ra4m1/soc.c#L78